.circleci/config.yml

```diff
@@ -228,8 +228,16 @@ jobs:
     <<: *test_template
 
   integration_tests_gatsby_source_wordpress:
-    executor: node
+    machine: true
```

Comment:
Trying to find some reference to this config setting and can't really find what `machine: true` does. I can see there is `machine` setting described in https://circleci.com/docs/2.0/configuration-reference/#machine but that seems to allow to pick an image (and not boolean switch).

What does it do actually?

Reply:
It runs that job in the default circleCI linux VM executor (https://circleci.com/docs/2.0/executor-types/#using-machine). We could also explicitly pick one but it didn't seem to matter too much since I'm using nvm to select a specific node version.
The reason for using this instead of the `node` executor that the other integration tests are using is that WP needs to run additional docker containers and circle doesn't allow networking between a custom executor (which runs in docker) and any additional containers for security reasons. When we set `machine: true` instead, it runs a linux VM which can properly network with  the WP docker containers that are spun up.